File: Day2/Solution.cpp
int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    float totalcost=0;
    float mealcost,tip,tax;
    cin>>mealcost>>tip>>tax;
    tip=tip*mealcost/100;
    tax=tax*mealcost/100;
    totalcost=mealcost+tip+tax;
    cout<<"The total meal cost is "<<round(totalcost)<<" dollars.";
    return 0;
}

File: Day2/Solution.py
mealCost = float(input())
tip = int(input())
tax = int(input())
tip=tip*mealCost/100;
tax=tax*mealCost/100;
totalcost=mealCost+tip+tax;

print ("The total meal cost is %s dollars." %str(int(round(totalcost, 0))))
